### ISSUE TITLE
Base Docker image for node runner

### ISSUE THREAD

**Issue by bard** [metrue/fx#517]
The node runner metrue/fx-node-base is currently built upon node:latest, which is in turn based upon Debian stretch, causing to brings in almost 1GB:

```
$ docker images|grep hello
hello                                           latest              0a0a40d809de        11 minutes ago      944MB
```

Would you consider switching to `node:alpine`? That is a much more manageable 117MB.

**Comment by metrue** [comment]
@bard 

Yes, it would be better,  welcome to PR ):

**Comment by metrue** [comment]
Fixed by #518
